src/styles/main.scss

```diff
@@ -1,3 +1,228 @@
-body {
+$color-action: #00acdc;
+$header-size: 60px;
+$color-white: #fff;
+$logo-size: 40px;
+$main-color: #060b35;
+$main-secondary-color: #616070;
+$star-size: 16px;
+
+@mixin square($size) {
+  width: $size;
+  height: $size;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+body,
+h2,
+h3 {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: $header-size;
+  background-color: $color-white;
+}
+
+.logo {
+  @include square($logo-size);
+}
+
+.nav {
+  list-style: none;
+
+  &__list {
+    padding: 0;
+    margin: 0;
+    display: flex;
+    justify-content: flex-end;
+    align-items: stretch;
+    align-content: center;
+  }
+
+  &__item {
+    display: flex;
+    margin-left: 20px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    color: $main-color;
+    text-transform: uppercase;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: $header-size;
+    white-space: nowrap;
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    &:hover {
+      color: $color-action;
+    }
+
+    &:link {
+      position: relative;
+      text-decoration: none;
+    }
+  }
+
+  .is-active {
+    color: $color-action;
+  }
+}
+
+.is-active:after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  width: 100%;
+  left: 0;
+  bottom: 0;
+  border-radius: 2px;
+  background-color: $color-action;
+}
+
+.main-catalog {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  width: 1024px;
+  padding: 50px 40px;
+  margin: 0 auto;
+}
+
+.card {
+  margin-right: 48px;
+  margin-bottom: 46px;
+  width: 200px;
+  background-color: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-color;
+    margin-top: 40px;
+  }
+
```

Comment:
You have magin collapse. The best practice is use only one way margin to avoid this (top or bottom and left or right)